service/src/main/java/io/camunda/service/UserServices.java

```diff
@@ -30,7 +30,7 @@ public UserServices(
       final CamundaSearchClient searchClient,
       final ServiceTransformers transformers,
       final Authentication authentication) {
-    super(brokerClient, searchClient, transformers, authentication);
+    super(brokerClient, searchClient, transformers, authentication, false);
```

Comment:
❓ I think we would want to apply authorizations to a user search query because I should only be able to see users if I am authorized to do so.

Reply:
same as above